lib/usd/pxrUsdPreviewSurface/usdPreviewSurface.h

```diff
@@ -76,6 +77,8 @@ class PxrMayaUsdPreviewSurface : public MPxNode
     PXRUSDPREVIEWSURFACE_API
     MStatus compute(const MPlug& plug, MDataBlock& dataBlock) override;
 
+    static MObject opacityThresholdAttr;
+
```

Comment:
As noted elsewhere, I don't think we want to expose this as a `public static` member.